rabbit-image/src/main/docker/rabbitmq.config

```diff
@@ -0,0 +1,10 @@
+[
+    {rabbit, [
+        {vm_memory_high_watermark, %memory_limit%},
+        {disk_free_limit, %disk_limit%}
+    ]},
+    {rabbitmq_management, [
+        {listener, [{port, 15672}]},
+        {load_definitions, "/etc/rabbitmq/rabbitmq_definitions.json"}
+    ]}
+].
```

Comment:
This controls the memory that RabbitMQ instances are allowed to use, sets the diskspace alarm amount, and configures an additional definitions .json file for providing a "high-availability" mirror configuration for queues.